### ISSUE TITLE
Build Python Classes/Types that represent fixed, finite lexicons of products and materials

### ISSUE THREAD

**Issue by RobertLRead** [PubInv/project-data#9]
Create a Python class to represent a fixed set of producible items, such as masks and cookies.

Create a Python class of a fixed set of items you might find in a Bill of Materials (BOM) such as butter, eggs, cotton fabric, NWPP, elastic.

**Comment by devhawk** [comment]
Example cookie recipe (found at random on internet) http://chasingjamesbeard.com/easy-oatmeal-chocolate-chip-cookie-recipe/

Consumable Ingredients Required:

* 1 cup butter, room temperature
* 1/2 cup granulated sugar
* 1/2 cup brown sugar
* 1 tsp vanilla extract
* 1 egg
* 1 cup flour
* 2 cups oatmeal
* 1.5 tsp baking soda 
* Salt
* 12 oz bag of semi-sweet chocolate chips
* parchment paper
  * included on list of ingredients since the parchment paper is not reusable
 
Reusable Tools required:
* Oven 
* mixer with mixing bowl
* baking sheet
* spoon

@RobertLRead this work for you?

**Comment by RobertLRead** [comment]
This is correct. In the Python code, I represent a "cookie" as an "elemental" product, unless I chose to model "butter" as an input. I that case, I would create "butter" as a new object in the supply chain, and assume that IT is "elemental", that is has no inputs.

As we discussed in the meeting, I think you decided to use the MakerMask OKH anyway. I THINK that has a clear distinction of tools in the OKH, but I did not attempt to model all of the inputs as consumables.

**Comment by devhawk** [comment]
Yup, going to use https://github.com/helpfulengineering/library/blob/main/alpha/okh/okh-manifest-makermask-origami.yml

**Comment by Jbutler-helpful** [comment]
Is this Resolved @devhawk
